### ISSUE TITLE
CSV export for object detection results

### ISSUE THREAD

**Issue by bartoszptak** [PUTvision/qgis-plugin-deepness#66]
Tool for exporting object detection results in CSV format. I think that the export button in the resulting dialog is a good place for it. 

Example CSV header:
```
class_name;confidence;longitude;latitude
```

**Comment by knwin** [comment]
@bartoszptak I agree with you. Even in the temporary output layer, these information should be included. then it can be exported to any GIS format. Normally I run my model with low confidence value and later I will filter the results with a desired threshold value so that I do not need to run the detection model with different confidence values.

**Comment by knwin** [comment]
I have updated the codes to add a confidence field in the attribute table of detection vector layer. Use lower confidence value in the GUI so that you can filter with your desired threshold in the result layer.

![Image](https://github.com/user-attachments/assets/ee6b612d-1561-46c0-829c-9ea448f6059b)

**Comment by przemyslaw-aszkowski** [comment]
Hi @knwin, I already merged your PR, it is on devel branch.

I can see the confidence of detections in a table if I right click "Open Attribute Table", but how can I see them in the Layers menu as on your screenshot?
This it how I can see it.

![Image](https://github.com/user-attachments/assets/7b443347-0cd5-49d9-b2b2-17d50c2f489e)

Thanks

**Comment by knwin** [comment]
It is just to use the graduated method in symbology.

**Comment by przemyslaw-aszkowski** [comment]
Thanks, works!
A note for others:

Right Click on the layer -> Properties -> Symbology -> "Graduated" in the top menu and "Value" set as "Confidence, Click "Classify" -> Apply

![Image](https://github.com/user-attachments/assets/f394825f-c12c-4782-b033-49c0bf0d445c)
